.github/workflows/ci.yml

```diff
@@ -27,18 +27,26 @@ jobs:
       CXX: clang++
       CC: clang
       PYTHON_BIN: /usr/bin/python3
+      # Install a specific version of Bazel that is supported by TensorFlow
+      # (see https://www.tensorflow.org/install/source#install_bazel).
+      BAZEL_VERSION: 1.1.0
     steps:
       - name: Installing bazel
         run: |
-          echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
-          sudo apt install curl
-          curl https://bazel.build/bazel-release.pub.gpg | sudo apt-key add -
-          sudo apt-get update && sudo apt-get install bazel
+          # https://docs.bazel.build/versions/master/install-ubuntu.html
+          sudo apt-get install unzip zip
+          wget https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
+          chmod +x bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
+          ./bazel-$BAZEL_VERSION-installer-linux-x86_64.sh --user
+          rm bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
```

Comment:
The whole VM gets killed after the job runs. Why bother deleting this file?

Reply:
No reason in particular. TensorFlow deletes it: https://github.com/tensorflow/tensorflow/blob/52d84416cbe8e30a58b0395985a7341fb8f96b1a/tensorflow/tools/ci_build/release/common.sh#L70-L75

We can keep the file around to simplify this config file.